node-services/cards-external-diffusion/src/tests/recapCardsDiffusionControl.test.ts

```diff
@@ -355,6 +356,75 @@ describe('Cards external diffusion', function () {
         expect(mailService.sent[0].body).toMatch(`Email Body Postfix`);
     });
 
+    it('Should send daily email with cards sorted by publishDate', async function () {
+        const time = Date.now();
+        const publishDateBeforeConfigDate = time - MILLISECONDS_IN_A_DAY / 2;
+        setup();
+        recapCardsDiffusionControl.setShowCardUrls(false);
+        initConfig();
+
+        opfabServicesInterfaceStub.usersWithPerimeters = [
+            {
+                userData: {login: 'operator_1', entities: ['ENTITY1']},
+                sendCardsByEmail: true,
+                emailForCardSending: 'operator_1@opfab.com',
+                computedPerimeters: perimeters
+            },
```

Comment:
Remove definition of operator_1 not needed in this test

Reply:
Removed, and also removed in other places in the file.